blocks/links-bar-block/features/links-bar/default.jsx

```diff
@@ -35,19 +34,23 @@ const LinksBar = ({ customFields: { navigationConfig = {}, ariaLabel } }) => {
 		<>
 			<nav
 				key={id}
-				className="links-bar"
+				className={BLOCK_CLASS_NAME}
 				aria-label={ariaLabel || phrases.t("links-bar-block.element-aria-label")}
 			>
 				{menuItems &&
 					menuItems.map((item, index) => (
-						<PrimaryFont as="span" className="links-menu" key={item._id}>
+						<span className={`${BLOCK_CLASS_NAME}__menu`} key={item._id}>
 							{item.node_type === "link" ? (
-								<Link href={item.url} name={item.display_name} />
+								<Link href={item.url} className={`${BLOCK_CLASS_NAME}__menu-link`} openInNewTab>
+									{item.display_name}
+								</Link>
 							) : (
-								<Link href={item._id} name={item.name} />
+								<Link href={item._id} className={`${BLOCK_CLASS_NAME}__menu-link`}>
+									{item.name}
+								</Link>
 							)}
 							{content.children.length !== index + 1 && showSeparator ? "\u00a0 • \u00a0" : ""}
```

Comment:
this can use the separator component in the arc themes components. this info is avilable in the figma https://www.figma.com/file/vWVGgZfZVBdSX7ACo3kmjr/Enhanced-Styling-Design-System?node-id=3507%3A34664 

https://github.com/WPMedia/arc-themes-components/tree/arc-themes-release-version-2.0.1/src/components/separato
<img width="821" alt="Screen Shot 2022-06-03 at 10 21 31" src="https://user-images.githubusercontent.com/5950956/171883675-c7141681-b06f-46d9-98b2-72c057f4ecbb.png">
r

Reply:
test